### PR TITLE
doc(youtube analytics): dbt model README edit for Fivetran style

### PR DESCRIPTION
**Pull request by fivetran-chloe**
Links https://fivetran.height.app/T-126487

Edits for Fivetran style. Rewords for clarity. Please address my question (inline) and check my edits for technical accuracy.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,18 +3,18 @@
 
 This package models YouTube Analytics data from [Fivetran's connector](https://fivetran.com/docs/applications/youtube-analytics#youtubeanalytics). It uses data in the format described by [this ERD](https://fivetran.com/docs/applications/youtube-analytics#schemainformation).
```
Comment (fivetran-chloe) [inline README.md:4]: YouTube Analytics doesn’t have an ERD. Can we rephrase this?

File: README.md [hunk 2]
```diff
@@ -49,19 +49,23 @@ vars:
...
 
-To alleviate this pain point, we have provided a solution which will allow you to create a [Fivetran Functions Connector](https://fivetran.com/docs/functions) that will sync your YouTube video metadata into a table in your warehouse. This dbt package can then use the `videos` metadata table to enrich your Fivetran YouTube Analytics reporting data. For more on how to create the Functions Connector, you can refer to our [YouTube Analytics Video Metadata Cloud Function](to_be_created_write-up_link_here) write up which 
+As a workaround, you can create a [Functions connector](https://fivetran.com/docs/functions) that syncs your YouTube video metadata into a table in your destination. This dbt package can then use the `VIDEOS` metadata table to enrich your YouTube Analytics reporting data. To learn more about creating a Functions connector, read our [YouTube Analytics Video Metadata Cloud Function article](to_be_created_write-up_link_here). It provides code and detailed steps on how to configure the function.
```
Comment (fivetran-chloe) [inline README.md:54]: Please update this link when the one-pager is ready

File: README.md [hunk 3]
```diff
@@ -29,7 +29,7 @@ packages:
 
 ## Configuration
 ## Required YouTube Channel Reports
-To use this package you will need to pull the following YouTube Analytics reports through Fivetran:
+To use this package, you need to pull the following YouTube Analytics reports through Fivetran:
 - [channel_basic_a2](https://developers.google.com/youtube/reporting/v1/reports/channel_reports#video-user-activity) (required)
 - [channel_demographics_a1](https://developers.google.com/youtube/reporting/v1/reports/channel_reports#video-viewer-demographics) (optional)
 - [videos metadata table](write-up_to_go_here) (optional)
```
Comment (fivetran-chloe) [inline README.md:35]: Please update this link when the one-pager is ready

File: README.md [hunk 4]
```diff
@@ -3,18 +3,18 @@
...
 | [youtube__gender_demographics_pivot](models/youtube__gender_demographics_pivot.sql)        | Each record represents a daily video view percentage with the gender options pivoted out for quicker analysis.            |
-| [youtube__video_metadata](models/youtube__gender_demographics_pivot.sql)           | Each record represents an individual video enriched with metadata. This model is disabled by default. Refer to our [YouTube Video Metadata Cloud Function Write Up](write-up_to_go_here) or our [YouTube Analytics source package](https://github.com/fivetran/dbt_youtube_analytics_source) for more details on how to generate this model.          |
+| [youtube__video_metadata](models/youtube__gender_demographics_pivot.sql)           | Each record represents an individual video enriched with metadata. This model is disabled by default. Read our [YouTube Video Metadata Cloud Function Write Up](write-up_to_go_here) or our [YouTube Analytics source package README](https://github.com/fivetran/dbt_youtube_analytics_source) to learn how to generate this model.          |
```
Comment (fivetran-chloe) [inline README.md:17]: Please update this link when the one-pager is ready

Comment (fivetran-joemarkiewicz) [reply README.md:17]: Will do, thanks!

File: README.md (hunk 1 above)
Comment (fivetran-joemarkiewicz) [reply README.md:4]: I think we phrased it best in the source pacakge 
```txt
It uses data in the format described by the [YouTube Channel Report schemas](https://fivetran.com/docs/applications/youtube-analytics#schemainformation).
```
I will update this section to reflect the source pacakge.

Comment (fivetran-chloe) [reply README.md:4]: Agreed - thanks for making the change!

**Review by fivetran-joemarkiewicz — APPROVED** [review]
Thanks again for the review! I will merge this with `main` and will be sure to update the video metadata write-up links once it is live!

**Pull request merged**